num2words/lang_TR.py

```diff
@@ -0,0 +1,575 @@
+# Copyright (c) 2017, Tufan Kaynak, Framras.  All Rights Reserved.
+
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+# MA 02110-1301 USA
+
+from __future__ import unicode_literals
+
+
+class Num2Word_TR(object):
+    def __init__(self):
+        self.precision = 2
+        self.negword = u"eksi "
+        self.pointword = u"virgül"
+        self.CURRENCY_UNIT = (u"lira",)
+        self.CURRENCY_SUBUNIT = (u"kuruş",)
+        # type(%s) not in [long, int, float]
+        self.errmsg_nonnum = u"Sadece sayılar yazıya çevrilebilir."
+        # Cannot treat float %s as ordinal.
+        self.errmsg_floatord = u"Tam sayı olmayan {} sıralamada kullanılamaz."
+        # Cannot treat negative num %s as ordinal.
+        self.errmsg_negord = u"Pozitif olmayan {} sıralamada kullanılamaz."
+        # abs(%s) must be less than %s.
+        self.errmsg_toobig = u"abs({}) sayı yazıya çevirmek için çok büyük. Yazıya çevrilebilecek en büyük rakam {}."
+        self.exclude_title = ["et", "virgule", "moins"]
+        # ordered number tuples in Turkish
+        self.ORDINAL_SIGN = (",",)
+        self.CARDINAL_BIRLER = (
+            u"sıfır", u"bir", u"iki", u"üç", u"dört", u"beş", u"altı", u"yedi", u"sekiz", u"dokuz",)
+        self.ORDINAL_BIRLER = (
+            u"", u"birinci", u"ikinci", u"üçüncü", u"dördüncü", u"beşinci", u"altıncı", u"yedinci", u"sekizinci", u"dokuzuncu",)
+        self.CARDINAL_ONLAR = (
+            u"", u"on", u"yirmi", u"otuz", u"kırk", u"elli", u"altmış", u"yetmiş", u"seksen", u"doksan",)
+        self.ORDINAL_ONLAR = (
+            u"", u"onuncu", u"yirminci", u"otuzuncu", u"kırkıncı", u"ellinci", u"altmışıncı", u"yetmişinci", u"sekseninci",
+            u"doksanıncı",)
+        self.CARDINAL_YUZLER = (u"yüz",)
+        self.ORDINAL_YUZLER = (u"yüzüncü",)
+        self.CARDINAL_UCLU_GRUPLAR = (u"", u"bin", u"milyon", u"milyar", u"trilyon", u"katrilyon", u"kentilyon",)
+        self.ORDINAL_UCLU_GRUPLAR = (
+            u"", u"bininci", u"milyonuncu", u"milyarıncı", u"trilyonuncu", u"katrilyonuncu", u"kentilyon",)
+        self.MAXVAL = (10 ** (len(self.CARDINAL_UCLU_GRUPLAR) * 3)) - 1
+
+        self.okunacak_tam_sayi = []
+        self.okunacak_uclu_grup_sayisi = 0
+        self.okunacak_artik_basamak_sayisi = 0
+        self.son_sifir_basamagi = 0
```

Comment:
Could you use English for var names, this will make easier the collaboration of future contributors?